mkosi

```diff
@@ -2493,7 +2545,8 @@ def install_grub(args: CommandLineArguments, workspace: str, loopdev: str, grub:
 
 def install_boot_loader_fedora(args: CommandLineArguments, workspace: str, loopdev: str) -> None:
     install_grub(args, workspace, loopdev, "grub2")
-
+def install_boot_loader_centos(args:CommandLineArguments, workspace: str, loopdev: str) -> None:
+    install_grub(args,workspace,loopdev,"grub2")
```

Comment:
please follow coding style, i.e.  spaces after comma...

But actually, given that this is the same function as  install_boot_loader_fedora() maybe just rename that one to install_boot_loader_and_centos()

Reply:
I think I managed to fix the messy commit history, I also made changes to try to align my changes with the coding style. I plan to make CentOS specific changes to the bootloader method so I think keeping them separate is best.